pipelines/frontend-pipeline.yml

```diff
@@ -0,0 +1,83 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - frontend/*
+      - pipelines/frontend-pipelines.yml
+    
+  pr:
+    autoCancel: true   
+    branches:
+      include:
+        - master
+    paths:
+      include:
+        - frontend/*
+        - pipelines/frontend-pipelines.yml
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  - group: prodoh-urlist-variable-group
+
+stages:
+  - stage: BuildStage
+    displayName: Build VUE Frontend
+    jobs:
+      - job: BuildJob
+        displayName: Build Test and Publish
+        steps:
+          - task: Npm@1
+            displayName: setup Npm
+            inputs:
+              command: 'install'
+              workingDir: './'
+          - task: Bash@3
+            displayName: build frontend
+            inputs:
+              targetType: 'inline'
+              script: 'npm run build'
+          - task: PublishBuildArtifacts@1
+            inputs:
+              PathtoPublish: 'dist'
+              ArtifactName: 'WebApp'
+              publishLocation: 'Container'
+  - stage: DeployStage
+    condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+    displayName: Deploying FE regionally
+    jobs:
+      - job: RegionalMaxtrixDeployment
+        displayName: Deploy FE to Regions
```

Comment:
Display name Suggestion:
Stage display name: Deploy
Job display name: Frontend to Regions